src/views/GameTemplates/GameTemplates.scss

```diff
@@ -0,0 +1,75 @@
+@import "../../index";
+
+.template-header {
+    display: flex;
+    color: $color-white;
+    font-size: 2rem;
+    justify-content: space-evenly;
+
+    .input-field {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-evenly;
+        padding: .5em;
+
+        input {
+            height: 50px;
+            width: 300px;
+            font-size: 1rem;
+            border: round;
+            border-color: white;
+            // border-radius: 10px;
+            box-shadow: 6px 6px 14px -5px rgba(0, 0, 0, 0.5);
+            background-color: transparent;
+            color: white;
+        }
+    }
+}
+
+.my-server1 {
+    display: flex;
+    align-items: center;
+
+    justify-content: space-between;
+
+    border-bottom: 1px solid #000;
```

Comment:
This is a minor change, but changing the color from #000 to #242424 makes the divider less pronounced

Reply:
Couldn't pick a color so I went with black just to have the line. This looks way better